components/script/dom/servohtmlparser.rs

```diff
@@ -112,9 +117,16 @@ impl AsyncResponseListener for ParserContext {
         match content_type {
             Some(ContentType(Mime(TopLevel::Image, _, _))) => {
                 self.is_synthesized_document = true;
-                let page = format!("<html><body><img src='{}' /></body></html>", self.url);
+                let page = format!("<html><body></body></html>");
```

Comment:
Nope, I just glanced at the block below and that one also has an unneeded `format!`, so I thought I'd stay consistent. I could change both, what would you prefer?